actionpack/lib/action_controller/railtie.rb

```diff
@@ -31,6 +32,10 @@ class Railtie < Rails::Railtie # :nodoc:
       ActionController::Helpers.helpers_path = app.helpers_paths
     end
 
+    initializer "action_controller.set_allowed_redirect_hosts" do |app|
+      config.action_controller.allowed_redirect_hosts << app.config.hosts
```

Comment:
This is a breaking change.
An application might have defined an old unused hostname in `config.hosts`.
With this change the redirects are allowed to that old hostname, without the developer knowing.

Reply:
You're right! I changed the code for not initializing this config with `app.config.hosts` 👍🏻